src/Security/DefenderForStorage.Autorest/README.md

```diff
@@ -0,0 +1,52 @@
+<!-- region Generated -->
+# Az.Security
+This directory contains the PowerShell module for the Security service.
+
+---
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.7.5 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.Security`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+###
+``` yaml
+commit: 6c4497e6b0aaad8127f2dd50fa8a29aaf68f24e6
+require:
+  - $(this-folder)/../../readme.azure.noprofile.md
+input-file:
+  - $(repo)/specification/security/resource-manager/Microsoft.Security/preview/2022-12-01-preview/defenderForStorageSettings.json
+
+title: Security
```

Comment:
The submodule name **Security** has been occupied by the other submodule in Az.Security. You may change it **DefenderForStorage**. As a result, you will need to make following code changes.

```
title: DefenderForStorage
root-module-name: $(prefix).Security
subject-prefix: Security
```

Reply:
Just making sure - 
Our service is under the Security RP (we are an extension RP, not an individual one), and should be under Az.Security submodule. During the office hour I had with the PWSH team, they asked me to create a new folder for our service (Defender for Storage), but said it'll still be under the Az.Security submodule. If I understand correctly, performing the change suggested here will cause it to be under a different, new, submodule, Az.DefenderForStorage. Am I right? If so, this is not the desired state for us.

Reply:
I think you misunderstood.  "it'll still be under the Az.Security submodule." should be something like "It is a submodule. It'll still be under Az.Security."

Reply:
I see, so even after the change, Az.Security will contain the new cmdlets of DefenderForStorage, but they'll just be encapsulated in the Az.DefenderForStorage submodule, right?

Reply:
Yes.